spec/models/manageiq/providers/inventory/persister/local_db_finders_spec.rb

```diff
@@ -20,7 +20,6 @@
                                :network_manager => FactoryBot.create(:ems_network, :zone => @zone))
 
     allow(@ems.class).to receive(:ems_type).and_return(:mock)
```

Comment:
can you also remove this as well?

Reply:
I'm assuming this line is to set what is returned from settings.

and settings[:unknown_key] probably is blank - that is why the second line is not needed.

Does seem like it would be better for this to just mock settings for the value expected to be passed into `:ems_cloud`

Reply:
Done.